gcc/config/riscv/riscv_vector.h

```diff
@@ -3487,6 +3487,20 @@ v##NAME##_f_f_w_f##SEW##m##LMUL##_m (					\
 _RVV_WFLOAT_ITERATOR_ARG (_RVV_FLOAT_NCVT_FF, nfcvt_ff, fncvt)
 _RVV_WFLOAT_ITERATOR_ARG (_RVV_FLOAT_NCVT_FF, nfcvt_rod_ff, fncvt_rod)
 
+#define _RVV_MASK_LOAD_STORE(MLEN, OP)					\
+__extension__ extern __inline vbool##MLEN##_t				\
+__attribute__ ((__always_inline__, __gnu_inline__, __artificial__))	\
+vlm_v_b##MLEN (const uint8_t *base, word_type vl)			\
+{									\
+  return __builtin_riscv_vlmbool##MLEN (base);				\
```

Comment:
`vlm`/`vsm` still need to set `VL` by `vsetvl*`, but I saw all other mask operation didn't handle that correctly too, I guess we need to fix that later, so I am OK without setting `VL` for now.